clusterloader2/pkg/measurement/common/slos/pod_startup_latency.go

```diff
@@ -121,16 +111,16 @@ func (p *podStartupLatencyMeasurement) start(c clientset.Interface) error {
 		klog.Infof("%s: pod startup latancy measurement already running", p)
 		return nil
 	}
-	p.selectorsString = measurementutil.CreateSelectorsString(p.namespace, p.labelSelector, p.fieldSelector)
+	p.selectorsString = p.selector.String()
```

Comment:
Is it necessary to keep the string? Cannot we just compute it every time it's needed?

Reply:
I'm not a fan of computing every time the same value - I prefer to store it.

Reply:
Please see the other comment - the same principle applies here. It's not used often enough to have any performance gain.

Reply:
selectorString removed.